portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,57 +34,99 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  public class Task {
+  public class Comment {
     public String commentInput;
     public long timestamp;
     public String name;
     public long id;
     public String email;
     public String displayemail;
 
-    public Task(long id, String name, String commentInput, long timestamp,  String email, String displayemail) {
+    public Comment(long id, String name, String commentInput, long timestamp,  String email, String displayemail) {
         this.commentInput = commentInput;
         this.timestamp = timestamp;
         this.name = name;
         this.id = id;
         this.email = email;
         this.displayemail = displayemail;
     }
+    public long getId() {
+      return id;
+    }
+    public String getTitle() {
+      return commentInput;
+    }
+    public long getTimestamp() {
+      return timestamp;
+    }
+    public String getName() {
+      return name;
+    }
+    public String getEmail() {
+      return email;
+    }
   }
-
+  public int maxcount = 3;
```

Comment:
Need more descriptive variable name here.  And this variable can be private, right?